### PR TITLE
Adjust Info Screen to announce new Release-Features for 2.11 (EXPOSUREAPP-9305)

### PR DESCRIPTION
**Pull request by jurajkusnier**
device-2021-09-20-155735

## Jira Ticket
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9305

### CODE REVIEW

File: Corona-Warn-App/src/main/res/values-de/release_info_strings.xml [hunk 1]
```diff
@@ -17,26 +17,35 @@
 
     <!-- XHED: Titles for the release info screen bullet points -->
     <string-array name="new_release_title">
-        <item>Information zu Auffrischungsimpfung</item>
-        <item>Druckversion von Impfzertifikaten</item>
+        <item>Universeller QR-Code Scanner</item>
```
Comment (InkenTownsend) [inline Corona-Warn-App/src/main/res/values-de/release_info_strings.xml:20]:
```suggestion
        <item>Universeller QR-Code-Scanner</item>
```

File: Corona-Warn-App/src/main/res/values-de/release_info_strings.xml [hunk 2]
```diff
@@ -17,26 +17,35 @@
...
+        <item>QR-Code importieren</item>
+        <item>Anpassung Kontaktdauer</item>
     </string-array>
 
     <!-- XTXT: Text bodies for the release info screen bullet points -->
     <string-array name="new_release_body">
-        <item>Die App kann Ihnen nun eine Mitteilung anzeigen, die Sie über eine eventuell anstehende Auffrischungsimpfung informiert.</item>
-        <item>Sie können nun eine Druckversion Ihrer Impfzertifikate anzeigen, die es Ihnen ermöglicht, die Zertifikate als PDF-Dokument zu speichern oder auszudrucken.</item>
+        <item>"Sie können nun unabhängig davon, von wo ein QR-Code Scanner in der CWA geöffnet wird, alle von der CWA unterstützen Dokumente scannen:
```
Comment (InkenTownsend) [inline Corona-Warn-App/src/main/res/values-de/release_info_strings.xml:27]:
```suggestion
        <item>"Sie können nun unabhängig davon, von wo ein QR-Code-Scanner in der CWA geöffnet wird, alle von der CWA unterstützen Dokumente scannen:
```

File: Corona-Warn-App/src/main/res/values-de/release_info_strings.xml [hunk 3]
```diff
@@ -17,26 +17,35 @@
...
+
+• PCR-Tests
+• Schnelltests
+• Event Check-ins
+• Zertifikate"</item>
+        <item>"Sie können nun auch QR-Codes von Fotos oder PDF-Dateien über den QR-Code Scanner importieren."</item>
```
Comment (InkenTownsend) [inline Corona-Warn-App/src/main/res/values-de/release_info_strings.xml:33]:
```suggestion
        <item>"Sie können nun auch QR-Codes von Fotos oder PDF-Dateien über den QR-Code-Scanner importieren."</item>
```

**Comment by InkenTownsend** [comment]
@jurajkusnier I realized that we spelled the QR-Code-Scanner with two - before and I changed it here to be consistent. Could you please take over this change? Thank you and sorry for that.

**Comment by jurajkusnier** [comment]
> @jurajkusnier I realized that we spelled the QR-Code-Scanner with two - before and I changed it here to be consistent. Could you please take over this change? Thank you and sorry for that.

no problem, it should be updated

**Review by InkenTownsend — APPROVED** [review]
Thank you. UA approved

**Review by mtwalli — APPROVED** [review]

**Review by axelherbstreith — APPROVED** [review]

**Pull request merged**